### PR TITLE
Add new third-party Go client to web service docs

### PR DESCRIPTION
**Pull request by oschwald**
Also remove previous Go client as it hasn't been updated since 2015.

### CODE REVIEW

File: content/geoip/docs/web-services.mdx [hunk 1]
```diff
@@ -106,8 +106,8 @@ and the Response Body section below.
     <tr>
       <td>Go</td>
       <td></td>
-      <td>[GoDoc](https://godoc.org/github.com/savaki/geoip2)</td>
-      <td>[GitHub](https://github.com/savaki/geoip2)</td>
+      <td>[Go Packages](https://pkg.go.dev/github.com/iamvladw/GeoIP2-go)</td>
+      <td>[GitHub](github.com/iamvladw/GeoIP2-go)</td>
```
Comment (horgh) [inline content/geoip/docs/web-services.mdx:110]: Should this link have a scheme?

**Review by horgh — CHANGES_REQUESTED** [review]

File: content/geoip/docs/web-services.mdx (hunk 1 above)
Comment (oschwald) [reply content/geoip/docs/web-services.mdx:110]: Gah, thanks!

Comment (oschwald) [reply content/geoip/docs/web-services.mdx]: I force-pushed the fix.

**Comment by oschwald** [comment]
I am going to close this as on further inspection, the new client seems a bit broken.

**Pull request closed without merging**